Jenkinsfile

```diff
@@ -81,7 +81,7 @@ void runParallelTests() {
 /* Run maven from tool "mvn" */
 void mvn(def args) {
   /* Get jdk tool. */
-  String jdktool = tool name: "jdk7", type: 'hudson.model.JDK'
+  String jdktool = tool name: "jdk8", type: 'hudson.model.JDK'
```

Comment:
Are you sure you want to switch to jdk8 for maven run?

The git client plugin must be compiled with jdk7.  The git plugin is compiled with jdk7 intentionally to reduce the risk of compatibility problems due to compiling with jdk8 but running on jdk7.